app/assets/javascripts/modules/FilterTable.js

```diff
@@ -45,34 +46,49 @@ define(['jquery', 'DoughBaseComponent', 'List'],
   };
 
   /**
-   * makeFilterTargetClasses
+   * toggleFilterWrappers function
    *
-   * Find columns with filterTargetClass and apply individual classes
-   * to them, since this is what ListJS wants.
+   * show/hide fitlerWrappers when all filterTargetClasses are hidden
    *
-   * @return {Array} array of classes used
    */
-  FilterTableProto.makeFilterTargetClasses = function () {
-    var filterClasses = [];
+  FilterTableProto.toggleFilterWrappers = function() {
+  	var self = this;
 
-    this.$el.find('.' + this.config.filterTargetClass).each(function(_, cell) {
-      var $cell = $(cell),
-          filterClass = 'js-filterable-' + $cell.index();
-      $cell.addClass(filterClass);
-      filterClasses.push(filterClass);
-    });
+		$(self.config.filterWrapperClass).each( function(index, element ) {
+			self.toggleFilterWrapper( $(element) )
+		});
+	}
```

Comment:
We can rename `index` to `_` — which is a convention that means 'this variable isn't going to be used'. We could also rename `element` to be something more indicative of what it is — like we would in an each block in ruby. `filterWrapper` I suppose?